### PR TITLE
feat: added storybook of templates 

### PR DESCRIPTION
**Pull request by maksha19**
[Notion link](https://www.notion.so/FE-storyBook-Templates-017f7e1e864b4ae4bf9fa126f5254e1e) 

This PR adds... 
-   Storybook screen of login, campaignLocation, customerDetails, dailyStatistics

- [x] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
- [x] I've requested reviews from 1-2 reviewers
- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
- [ ] I've added/updated unit/integration tests
- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3) for at least the happy flow
- [ ] I've added [Chinese translations for i18n setup](https://www.notion.so/Translations-Dev-Guide-8c1da838982a4f59a386ec96ac6780c8)
- [ ] I've added documentation in README/[Notion](https://www.notion.so/82b92fb1007640328dab9582c0a3694e?v=3b6ce48202cf40ad8450553799b13146)

### CODE REVIEW

File: storybook/stories/CampaignLocationScreen/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+storiesOf("Screen", module)
+  .addDecorator((Story: any) => (
+    <AuthStoreContext.Provider
+      value={{
+        hasLoadedFromStore: true,
+        authCredentials: {
+          "Campaign 1": {
+            operatorToken: "operatorToken",
+            sessionToken: "sessionToken",
+            expiry: new Date().setDate(new Date().getDate() + 1),
```
Comment (andrehadianto) [inline storybook/stories/CampaignLocationScreen/index.tsx:22]: Can replace with this instead
```suggestion
            expiry: Date.now() + 86400000 // 1 day in millisecond,
```

File: storybook/stories/CampaignLocationScreen/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            endpoint: "Endpoint",
+          },
+          "Campaign 2": {
```
Comment (andrehadianto) [inline storybook/stories/CampaignLocationScreen/index.tsx:25]: Somehow shows as `Campaign Name` on my screen? Not sure if this is intended

File: storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+    categoryType: "APPEAL",
+    name: "Damaged token",
+    order: 3,
+    quantity: {
+      period: 7,
+      limit: 2,
+    },
+  },
+];
+
```
Comment (andrehadianto) [inline storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx:45]: Can add another `category: "Defective token"` to match the current appeal options!

File: storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+import { navigation, mockReactNavigationDecorator } from "../mocks/navigation";
+import { provider as MockProvider } from "../mocks/provider";
+
+const products: CampaignPolicy[] = [
+  {
+    category: "Lost token",
+    categoryType: "APPEAL",
+    name: "Lost token",
+    alert: {
+      label: "*chargeable",
```
Comment (andrehadianto) [inline storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx:15]: I can't seem to see the `*chargeable` label, is this intended?

File: storybook/stories/DailyStatisticsScreen/index.tsx [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+    )
+  );
+  return <Navigator />;
+};
+
+storiesOf("Screen", module)
+  .addDecorator((Story: any) => reactNavigationDecorator(Story))
+  .add("DailyStatistics", () => (
+    <View style={{ height: "100%" }}>
+      <DailyStatisticsScreenContainer
```
Comment (andrehadianto) [inline storybook/stories/DailyStatisticsScreen/index.tsx:47]: Is it possible to add some sample `pastTransactions`?

**Review by andrehadianto — CHANGES_REQUESTED** [review]
Have some comments!

File: storybook/stories/CampaignLocationScreen/index.tsx (hunk 1 above)
Comment (maksha19) [reply storybook/stories/CampaignLocationScreen/index.tsx:22]: thank you. will update like that

File: storybook/stories/DailyStatisticsScreen/index.tsx (hunk 5 above)
Comment (maksha19) [reply storybook/stories/DailyStatisticsScreen/index.tsx:47]: this one need data from BE. the mocking of data will be in another PR.

File: storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx (hunk 4 above)
Comment (maksha19) [reply storybook/stories/CustomerAppeal/CustomerAppealScreen.tsx:15]: in order to show. need to mock  data.  the mock of API service will be handle it different PR.

**Review by andrehadianto — APPROVED** [review]
LGTM!

**Pull request merged**